src/main/java/seedu/address/logic/commands/ViewCommand.java

```diff
@@ -21,10 +21,17 @@ public class ViewCommand extends Command {
     public static final String MESSAGE_NO_SUCH_TELEGRAM =
```

Comment:
There is the variable `MESSAGE_INVALID_TELEGRAM = "Cannot find member(s) with telegram handle: %1$s";` in `Message.java` file, may be better to use that instead of duplicate one error message?